homepairs/homepairsUI/tests/redux/account/FetchAccount.test.tsx

```diff
@@ -260,10 +272,11 @@ describe('FetchAccount Action', () => {
         mock.onPost('https://homepairs-alpha.herokuapp.com/API/login/').reply(200, data);
         await propertyManagerMock1.dispatch(AccountActions.fetchAccount(email, password, testProps.navigation, statusFailedSpy)).then(() => {
           expect(propertyManagerMock1.getActions()).toHaveLength(0);
-          expect(statusFailedSpy.mock.calls).toHaveLength(1)
+          expect(statusFailedSpy.mock.calls).toHaveLength(1);
```

Comment:
Should we check the specific failure at all? (not just the length)

Reply:
We could but we haven't generated all of our error messages yet. This is something Cesar can test. It would literally be a waste of time for me to do this at this point.

Reply:
sounds good, I agree

Reply:
hmmm I don't handle these error messages tho. Not with what I've made at this point. I don't handle errors from the API calls.